bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -146,32 +149,40 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 }
                 break;
             case RenaultBindingConstants.CHANNEL_HVAC_STATUS:
-                // We can only trigger pre-conditioning of the car.
-                if (command instanceof StringType && command.toString().equals(Car.HVAC_STATUS_ON)
-                        && !car.isDisableHvac()) {
-                    final MyRenaultHttpSession httpSession = new MyRenaultHttpSession(this.config, httpClient);
-                    try {
-                        updateState(CHANNEL_HVAC_STATUS, new StringType(Car.HVAC_STATUS_PENDING));
-                        car.resetHVACStatus();
-                        httpSession.initSesssion(car);
-                        httpSession.actionHvacOn(car.getHvacTargetTemperature());
-                        if (pollingJob != null) {
-                            pollingJob.cancel(true);
+                if (!car.isDisableHvac()) {
```

Comment:
Can the HVAC enabled/disabled state change after polling new data? In this case `RefreshType` should be checked unconditionally.

Reply:
No. Features are switched off (if the service is forbidden etc..) and then is skipped until the thing is deactivated and reactivated.